core/settings.py

```diff
@@ -114,6 +115,27 @@ class Common(Configuration):
     BROKER_PORT = os.getenv("BROKER_PORT", "5672")
     BROKER_URL = f"amqp://{BROKER_HOST}:{BROKER_PORT}"
 
+    REST_FRAMEWORK = {
+        'DEFAULT_PERMISSION_CLASSES': (
+            'rest_framework.permissions.IsAuthenticated',
+        ),
+        'DEFAULT_AUTHENTICATION_CLASSES': (
+            'rest_framework.authentication.SessionAuthentication',
+            'rest_framework_simplejwt.authentication.JWTAuthentication',
+        )
+    }
+
+    ACCESS_TOKEN_LIFETIME_IN_MINUTES = int(
+        os.getenv('ACCESS_TOKEN_LIFETIME_IN_MINUTES', 1440)
```

Comment:
Pode colocar um comentário indicando que seriam 24 horas?

```suggestion
        os.getenv('ACCESS_TOKEN_LIFETIME_IN_MINUTES', 1440)  # 24 horas
```